.ci/Jenkinsfile

```diff
@@ -61,6 +61,7 @@ pipeline {
            dir("${BASE_DIR}"){
               setEnvVar('BEAT_VERSION', sh(label: 'Get beat version', script: 'make get-version', returnStdout: true)?.trim())
               log(level: 'INFO', text: "env.BEAT_VERSION=${env.BEAT_VERSION}")
+              cmd(label: 'check', script: 'make -C deploy/kubernetes all')
```

Comment:
I'd say to move this to the k8s specific stage, see `L153`, the k8s stage runs only if there are changes matching the regex `(^deploy/kubernetes/.*|^version/docs/version.asciidoc)` (see L54) on a PR basis or by default on branches

Reply:
OTOH, what's the difference between `make -C deploy/kubernetes all` and `make -C deploy/kubernetes test`, since `test` is already in place.

Reply:
`make -C deploy/kubernetes all` generates the kube*.yml files after which a `MAKE check-no-changes` should be ran to make sure the templates are up to date. This step should take place in the Check area

Reply:
What do you think if:
1) `make -C deploy/kubernetes all` could be renamed to a different make goal, so the name of the goal reflects what it does.
2) change `make check-ci` to run the above make goal.


https://github.com/elastic/elastic-agent/blob/63b682fefc61d11d0976542be6d2e168e4efbf94/Makefile#L44-L48

With the above, there is no need to change the pipeline, there is only one make goal to run the checks either locally or in the CI, and additionally the make goal `all` could reflect what it does

What do you think?

Reply:
I have no issues with that option as well, made the necessary changes